src/database/list/loaded.ts

```diff
@@ -0,0 +1,57 @@
+import { DatabaseQuery, ChildEvent, DatabaseSnapshot } from '../interfaces';
+import { fromRef } from '../observable/fromRef';
+import { createListSnapshotChanges } from './snapshot-changes';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/skipUntil';
+import 'rxjs/add/operator/skipWhile';
+import 'rxjs/add/operator/withLatestFrom';
+import 'rxjs/add/operator/do';
+import 'rxjs/add/operator/filter';
```

Comment:
Refactor these to `call` style?
such as mentioned here https://github.com/angular/material2/blob/c564df61f48ff9c40eea237c394a15adac19cfdb/CODING_STANDARDS.md#rxjs

Reply:
Yep! Totally on the `//TODO:  FIX THIS` list. Wanted to get the POC up and running first.

Reply:
Yeah no probs, just being nit picky

Reply:
Comments are always welcome :)